FutureNorthants/northants-website
# Backend support for homepage hero logo image override

### Setup
1. Checkout this branch. It's based on the summary PR to keep config changes down.
2. `./drush.wrapper @docker config:import`
3. Clear the drupal cache

### Testing
1. In the CMS, view the homepage. Note lovely new preview layout.
2. Edit the homepage and set a hero logo image override and save/publish. View the page and note lovely preview now includes lovely overridden logo.
3. In the /graphql form use the following query:
```
{
  route(path: "/") {
    node {
      title
      heroLogoOverride {
        url
        altText
      }
    }
  }
}
```
...and verify the URL and altText are correct.
4. Edit the homepage, remove the logo override and save/publish. Rerun the graphql query and verify that heroLogoOverride now returns null.
